packages/icons/scripts/web.ts

```diff
@@ -85,7 +85,7 @@ function buildComponentFile(component: Component): string {
       ${sizeMap.join('\n')}
     }
 
-    export const ${component.name} = ({ size = 'small', ...props }) => {
+    export const ${component.name} = ({ size = '24', ...props }) => {
```

Comment:
This is where we're destructuring the `size`. Since we're only spreading `props`, it should not be in the DOM 🤔

Reply:
This code is only used when multiple sizes are available. If there's only one size, the code on L69-L71 is used. I'd suggest destructuring the size there, too.

Reply:
👀 nice catch, that's why

Reply:
Is this what you. had in mind? 3ec788e

Reply:
Yes, that's exactly it.